.github/workflows/proto-registry.yml

```
@@ -0,0 +1,92 @@
+name: Buf Push
```

The name and filename don't need to be exact matches, but it's good to have them in sync.
```suggestion
name: Third Party Proto Registry
```
See other comments on why "Third Party" is in there.

Basically, you want it to be easy to start at Github, looking at the results of a workflow, and know which file to edit locally. Slightly less important is knowing the filename and wanting to find a run of the workflow, but it does happen.